content/client-server-api/_index.md

```diff
@@ -2540,6 +2540,9 @@ return a standard error response of the form:
 The `retry_after_ms` key SHOULD be included to tell the client how long
 they have to wait in milliseconds before they can try again.
 
+The `Retry-After` HTTP header MAY be specified by the server. When this header
```

Comment:
My reading of the MSC is rather stronger than this MAY wording. It actually uses the word "shall". In any case, it seems like compliant homeservers now *need* to include the header. I'd also like us to explicitly use the word "deprecated" for the old property.

Reply:
Adjusted.